api/v1beta2/moduletemplate_types.go

```diff
@@ -138,11 +138,34 @@ type ModuleTemplateSpec struct {
 
 	// Info contains metadata about the module.
 	// +optional
-	Info ModuleInfo `json:"info,omitempty"`
+	Info *ModuleInfo `json:"info,omitempty"`
 
 	// AssociatedResources is a list of module related resources that usually must be cleaned when uninstalling a module. Informational purpose only.
 	// +optional
 	AssociatedResources []apimetav1.GroupVersionKind `json:"associatedResources,omitempty"`
+
+	// Manager contains information identifying a module's resource that indicates the readiness of the module. Typically, this should be the manager Deployment of the module.
```

Comment:
```suggestion
	// Manager contains information for identifying a module's resource that can be used as indicator for the installation readiness of the module. Typically, this is the manager Deployment of the module. In exceptional cases, it may also be another resource. 
```